src/test/java/org/jabref/logic/importer/fetcher/ResearchGateTest.java

```diff
@@ -119,6 +119,7 @@ void performSearchWithBibEntry() throws FetcherException {
     }
 
     @Test
+    @DisabledOnCIServer("CI server is unreliable")
     void performSearchWithTitleWithCurlyBraces() throws FetcherException {
         BibEntry entryInput = new BibEntry()
```

Comment:
Easier if you also set the StandardEntryType Misc here and then in the assertEquals just compare both entries